docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md

```diff
@@ -0,0 +1,351 @@
+---
+title: "Use service-to-service correlation in Azure Service Bus solutions"
+layout: default
+---
+
+# Use service-to-service correlation in Azure Service Bus solutions
+The concept of service-to-service correlation is big and complex and spans multiple Arcus libraries. This user-guide will walk through all the available Arcus features that work together to facilitate service-to-service correlation in your Azure Service Bus solutions. For a general introduction, see the [introduction page on service-to-service correlation](index.md).
+
+To use service-to-service correlation in Azure Service Bus solutions, both the sending and receiving side of the separate components have to be adapted. The correlation will be passed through from one service to another via the Azure Service Bus message application properties which the internal Arcus system will use to link your services together.
```

Comment:
```suggestion
To use service-to-service correlation in Azure Service Bus solutions, both the sending and receiving side of the separate components have to be adapted. The correlation will be passed through from one service to another via the Azure Service Bus message application properties which Arcus will use to link your services together.
```
Isn't that simpler / easier to read ?

Reply:
Yes, that's true. Thx! 👍 
https://github.com/arcus-azure/arcus.observability/pull/467/commits/7afadc0cf3ab390f9fe62845d5cd1ff0abdc8e02